extensions/VisualEditor/ApiVisualEditor.php

```diff
@@ -359,6 +369,10 @@ public function execute() {
 			$parserParams['oldid'] = $params['oldid'];
 		}
 
+		if ( isset( $params['preload'] ) ) {
+			$parserParams['preload'] = $params['preload'];
```

Comment:
"preload" is not a parser parameter so should not be added to $parserParams array